app/models/concerns/profile_searching.rb

```diff
@@ -94,6 +100,8 @@ module ProfileSearching
 
   # class methods for profile searching
   module ClassMethods
+    include ScopedSearch
```

Comment:
Is this needed because of the `scoped_search_definition`? I'm a bit worried about the side effects of including such a big module, isn't there any other way to refer these methods?

Reply:
It actually worked without this. I'll remove it :slightly_smiling_face: